dev-libs/device-atlas-api-c/device-atlas-api-c-2.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit unpacker cmake-multilib
+
+MY_P="deviceatlas-enterprise-c-${PV}"
+
+DESCRIPTION="API to detect devices based on the User-Agent HTTP header"
+HOMEPAGE="https://deviceatlas.com"
+SRC_URI="https://deviceatlas.com/download/api/enterprise/c -> ${MY_P}.zip"
+
+LICENSE="public-domain"
```

Comment:
As a registration to the website is the only requirement to get the source code as direct redistribution is not allowed, would it be possible to get either this a custom licence text

> License
> 
> The free of charge offering has a restricted use license, for own usage only; no redistribution or derivative works are permitted. Contact https://www.deviceatlas.com/contact-us to upgrade or to enable usage as part of a service offering.

or any alternative ?

Thanks.